src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ConnectionString.cs

```diff
@@ -20,6 +20,8 @@ public class ConnectionString
         private Dictionary<string, string> _keyValuePairs;
         private string _connString;
         private StringBuilder _parseErrorSb;
+        private string DEFAULT_TENANTID = "72f988bf-86f1-41af-91ab-2d7cd011db47";
```

Comment:
Need to update the version number in prject.json so we can publish this for PowerShell.  Also, how hard would it be to add a test.